new parse_fraction function for simple fractions added with test data (#60)
scrapinghub/number-parser: README.rst

Usage
=====

The library provides the following common usages.

> --------
> - **Converting numbers in-place**
> --------

Reviewer: I think only the text should change, the formatting should remain the same, in line with the rest of the file:

```suggestion
Converting numbers in-place
---------------------------
```

The same applies to the other headers.

Author: Sorry for that, I have changed the formatting so that it will be in line with the rest of the file.

>>> parse("First day of year two thousand")
'1 day of year 2000'

> Parsing a number
> ---------------------------

Reviewer: ```suggestion
Parsing a number
----------------
```

Looks slightly better, and otherwise Sphinx logs a warning, I believe.

Author: Thanks for pointing it out, I have corrected it.:smile:

>>> parse_ordinal("seventy fifth")
75

Parsing a fraction
------------------

> Converting a fractional number written in words to it's corresponding integeral fraction. (Support for only English)

Reviewer: ```suggestion
Converting a fractional number written in words to its corresponding integral fraction. (Support for English only)
```

Author: Thanks, I have corrected these mistakes.
